cloudflare/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:jessie
```

Comment:
Given that this is a container for managing DNS, I didn't want any DNS-related surprises from musl